docs/source/en/main_classes/logging.md

```diff
@@ -30,7 +30,7 @@ transformers.logging.set_verbosity_info()
 ```
 
 You can also use the environment variable `TRANSFORMERS_VERBOSITY` to override the default verbosity. You can set it
-to one of the following: `debug`, `info`, `warning`, `error`, `critical`. For example:
+to one of the following: `debug`, `info`, `warning`, `error`, `critical`, `fatal`. For example:
```

Comment:
I don't think we have `fatal`, where is this coming from?

Reply:
https://github.com/huggingface/transformers/blob/a70286f827ea46573e498fbc185a2498bc67d709/src/transformers/utils/logging.py#L168-L178

Currently using `transformers-4.44.2` in a Kaggle Notebook, I am able to generate the logs with `logger.fatal("FATAL")`.

![image](https://github.com/user-attachments/assets/9339527b-cebf-4057-9335-a81b7af634b4)

Is there any plan to deprecate this functionality? If not, then I apologize for the confusion, I didnt see that `critical` and `fatal` have the same logging level and the fixed commit should have been:
`debug, info, warning, error, critical/fatal`

If `fatal` is going to be deprecated, i'll update the PR and fix it, do let me know if that's the case. Thanks for Reviewing 🤗

Reply:
No worries, let's keep it like that in the meantime